shell/platform/linux/fl_view.cc

```diff
@@ -500,6 +510,28 @@ static gboolean event_box_motion_notify_event(GtkWidget* widget,
   return TRUE;
 }
 
+static gboolean event_box_leave_notify_event(GtkWidget* widget,
+                                             GdkEventCrossing* event,
+                                             FlView* view) {
+  if (view->engine == nullptr) {
+    return FALSE;
+  }
+
+  // Don't remove pointer while button is down; In case of dragging outside of
+  // window with mouse grab active Gtk will send another leave notify on
+  // release.
+  if (view->pointer_inside && view->button_state == 0) {
+    gint scale_factor = gtk_widget_get_scale_factor(GTK_WIDGET(view));
+    fl_engine_send_mouse_pointer_event(
+        view->engine, kRemove, event->time * kMicrosecondsPerMillisecond,
```

Comment:
We don't currently send a kAdd so this event is not symmetrical. The current code seems to work fine, but I wonder if we should also do the kAdd? I guess that would go into `event_box_enter_notify_event()`.